### ISSUE TITLE
monaco-editor release on cdnjs.org is at least a month out of date

### ISSUE THREAD

**Issue by AtOMiCNebula** [microsoft/monaco-editor#1780]
[cdnjs.org hosts monaco-editor](https://cdnjs.com/libraries/monaco-editor), and I leverage that to integrate it with an ASP.NET Core project of mine ([libman](https://docs.microsoft.com/en-us/aspnet/core/client-side/libman/libman-vs?view=aspnetcore-3.1)'s integration with it is very nice).  However, as of this writing, the most recent available release on cdnjs.org is v0.18.1, while the most current release on GitHub is v0.19.3.  v0.19.0 was tagged on December 20th, leaving cdnjs.org more than a month out of date.

It looks like their auto-updater has broken and is no longer picking up new versions that are posted to npm.  Is this something the Monaco team would know how to resolve, or be interested in resolving?

**Comment by rebornix** [comment]
Not sure what we should do here as it seems cdnjs picks up npm packages automatically.

**Comment by AtOMiCNebula** [comment]
I looked into this a bit more, and yeah CDNJS *should* be picking things up automatically (but clearly haven't been).  Based on their issue tracker, this doesn't appear to be an isolated issue with them.  Their ingestion process seems not nearly as automated as it should be.

I ended up moving my app away from CDNJS to jsDelivr, which live-mirrors npm (and it can be used with libman too).  Sorry for the noise here!
